woocommerce/woocommerce-android
# Product creation using package photo: Analytics

This PR adds the event tracking to the AI product generation using package photo feature.

**To test:**
1. Go to Products
2. Tap on + button
3. Select the AI option
4. Tap on the Use package photo button
5. Select a photo with text from the media picker
6. Wait for the name/description generation to complete
7. Uncheck some keywords and tap on the Regenerate button
8. Tap on the Continue button
9. Verify the following events were recorded:
```
🔵 Tracked: product_name_ai_package_image_button_tapped
🔵 Tracked: media_picker_opened
🔵 Tracked: media_picker_item_selected
🔵 Tracked: add_product_from_image_displayed
🔵 Tracked: add_product_from_image_scan_completed, Properties: {"scanned_text_count”:X,…}
🔵 Tracked: ai_identify_language_success, Properties: {"language”:”xx”,"source":"product_creation_from_package_photo”…}
🔵 Tracked: add_product_from_image_details_generated, Properties: {"language”:”xx”,”selected_text_count”:X,…}
🔵 Tracked: add_product_from_image_regenerate_button_tapped
🔵 Tracked: ai_identify_language_success, Properties: {"language”:”xx”,"source":"product_creation_from_package_photo”…}
🔵 Tracked: add_product_from_image_details_generated, Properties: {"language”:”xx”,”selected_text_count”:X,…}
🔵 Tracked: add_product_from_image_continue_button_tapped
```
